python/sbp/client/loggers/json_logger.py

```diff
@@ -76,11 +72,9 @@ def next(self):
     for line in self.handle:
       try:
         data = json.loads(line)
-        delta = data['delta']
-        timestamp = data['timestamp']
-        item = SBP.from_json_dict(data['data'])
+        item = SBP.from_json_dict(data.pop('data'))
         msg = self.dispatch(item, line)
-        yield (delta, timestamp, msg)
+        yield (msg, data)
```

Comment:
It's a tuple of a message and a dictionary of metadata.  The same as produced by Framer and Handler iterators.